packages/polkadart_scale_codec/test/test_types/signed_int_test.dart

```diff
@@ -607,4 +633,386 @@ void main() {
       });
     });
   });
+
+  group('decode from hex and return signed int tests', () {
+    final typesRegistry = <String, dynamic>{
+      'Codec': {
+        'a': 'i8',
+        'b': 'i16',
+        'c': 'i32',
+        'd': 'i64',
+        'e': 'i128',
+        'f': 'i256',
+      },
+    };
+    // Creates the registry for parsing the types and selecting particular schema.
+    final registry = OldTypeRegistry(types: typesRegistry);
+
+    // specifyng which schema type to use
+    registry.select('Codec');
+
+    // fetching the parsed types from `Json` to `Type`
+    final types = registry.getTypes();
+
+    // Initializing Scale-Codec object
+    final codec = Codec(types);
+
+    group('fixed-length 8bit', () {
+      test('Given an encoded string `0x45` it should return 69', () {
```

Comment:
please don't use magic numbers and properly describe the scenario that you're testing
```suggestion
      test('Given an encoded string when it represents a positive integer it should be decoded', () {
```